js/samples/ios/quickstart-swift/ImmersiveReaderViewController.swift

```diff
@@ -31,39 +30,38 @@ public class ImmersiveReaderViewController: UIViewController, WKUIDelegate, WKNa
         self.onFailureImmersiveReader = onFailureImmersiveReader
         self.onTimeout = onTimeout
         self.onError = onError
+        self.src = "https://" + subdomainToPass + ".cognitiveservices.azure.com/immersivereader/webapp/v1.0/reader"
         super.init(nibName: nil, bundle: nil)
     }
-    
+
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-    
+
     override public func viewDidLoad() {
         super.viewDidLoad()
 
         // If uiLang options are set update src to reflect this.
         switch optionsToSend?.uiLang {
         case .none: break
-            // Leave src as is.
         case .some(let value):
             src = src + "?omkt=" + value
         }
-        
+
         // Set timeout to default or value user specifies.
         switch optionsToSend?.timeout {
         case .none:
-            // Default to 15 seconds.
             timeoutValue = 15
         case .some(let value):
             timeoutValue = value
         }
-      
+
         view.backgroundColor = .white
         webView = WKWebView()
-        
+
         let contentController = WKUserContentController()
         if #available(iOS 11.0, *) {
-            webView = ImmersiveReaderWebView(frame: .zero, contentController: contentController)
+            webView = WKWebView(frame: .zero, contentController: contentController)
```

Comment:
This should be the configuration parameter, there's no contentContoller param in WKWebView.